src/server/raft_client.rs

```diff
@@ -428,6 +432,7 @@ struct AsyncRaftSender<R, M, B, E> {
     buffer: B,
     router: R,
     snap_scheduler: Scheduler<SnapTask>,
+    tablet_scheduler: Scheduler<SnapTask>,
```

Comment:
I think we can only use one scheduler，only need exist one.

Reply:
yes, the two scheduler should not work in one time.